lib/app/client.js

```diff
@@ -384,8 +386,30 @@ async function render (to, from, next) {
           promises.push(p)
       }
 
-      return Promise.all(promises)
-    }))
+      return allPromisesHandled(promises).catch(async (errors) => 
```

Comment:
No need to async function. We can Simply `return Promise.resolve/reject` to chain them.

Reply:
That's true, I'll remove it in a new commit.